software/arduino/FastLedManager.cpp

```diff
@@ -1,11 +1,11 @@
 #include "FastLedManager.hpp"
 
-FastLedManager::FastLedManager(const uint8_t numLeds, DeviceType deviceType,
+FastLedManager::FastLedManager(DeviceDescription *device,
```

Comment:
The LED manager should never modify the DeviceDescription, right? Can this be `DeviceDescription const *const device`?

Reply:
I was reading and I think `const DeviceDescription *device` makes more sense because the pointer should be copied when passed in thus the second `const` is unnecessary. Furthermore Google C++ style _prefers_ `const` to the left of the type.

Reply:
SGTM